packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -34,6 +34,26 @@ import 'target_devices.dart';
 
 export '../cache.dart' show DevelopmentArtifact;
 
+abstract class DotEnvRegex {
+  // Dot env multi-line block value regex
+  static final RegExp block = RegExp(r'^\s*([a-zA-Z_]+[a-zA-Z0-9_]*)\s*=\s*"""\s*(.*)$');
+
+  // Dot env full line value regex (eg FOO=bar)
+  static final RegExp property = RegExp(r'^\s*([a-zA-Z_]+[a-zA-Z0-9_]*)\s*=\s*(.*)?$');
+
+  // Dot env value wrapped in double quotes regex (eg FOO="bar")
+  static final RegExp doubleQuoteValue = RegExp(r'^"(.*)"\s*(\#\s*.*)?$');
+
+  // Dot env value wrapped in single quotes regex (eg FOO='bar')
+  static final RegExp quoteValue = RegExp(r"^'(.*)'\s*(\#\s*.*)?$");
+
+  // Dot env value wrapped in back quotes regex (eg FOO=`bar`)
+  static final RegExp backQuoteValue = RegExp(r'^`(.*)`\s*(\#\s*.*)?$');
+
+  // Dot env value without quotes regex (eg FOO=bar)
+  static final RegExp noQuoteValue = RegExp(r'^([^#\n\s]*)\s*(?:\s*#\s*(.*))?$');
```

Comment:
It is not obvious what the difference between the `property` and `noQuoteValue` regexes are. I can guess that `property` refers to any kind of entry, while `noQuoteValue` refers specifically to entries where the value is not quoted. Is there a way to make the difference more obvious both with the name and the comment (which uses the same example)?

Reply:
I agree. Naming is not obvious enough. I will try to fix it today.

Reply:
I pushed a new version.

Variables have been renamed to be more explicit.
I also added new comment lines to precise which string will be matched

Reply:
Thanks!

Reply:
You're welcome